lib/features/reps_core/reps_admin/reps_admin.module

```diff
@@ -413,3 +413,22 @@ function reps_admin_theme_registry_alter(&$theme_registry) {
     }
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function reps_admin_form_alter(&$form, &$form_state, $form_id) {
```

Comment:
here you need to implement hook_form_FORM_ID_alter so reps_admin_form_views_exposed_form_alter

Reply:
ok